build.gradle

```diff
@@ -15,10 +15,8 @@ buildscript {
     }
 }
 
-apply plugin: 'application'
 apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
```

Comment:
'org.springframework.boot' should be in application dependencies and it  implies 'io.spring.dependency-management', thus this one is not needed at all.